_includes/specials/policy/en/03-bidding-for-block-space.md

```diff
@@ -0,0 +1,105 @@
+<!--
+  300 to 1000 words
+  put title in main newsletter
+  put links in this file
+  for any subheads use h3 (i.e., ###)
+  illustrations welcome (max width 800px)
+  if uncertain about anything, just do what seems best and harding will edit
+-->
+
+Last week we mentioned that transactions pay fees for the used
+blockspace rather than the transferred amount, and established that
+miners optimize their transaction selection to maximize collected fees.
+It follows that only those transactions get confirmed that reside in the
+top of the mempool when a block is found. In this post, we will discuss
+practical strategies to get the most for our fees. Let’s assume we have
+a decent source of feerate estimates—we will talk more about feerate
+estimation in next week’s article.
+
+While constructing transactions, some parts of the transaction are more
+flexible than others. Every transaction requires the header fields, the
+recipient outputs are determined by the payments being made, and most
+transactions require a change output. Both sender and receiver should
+prefer blockspace-efficient output types to reduce the future cost of
+spending their transaction outputs, but it’s during the [input
+selection][topic coin selection] that there is the most room to change
+the final composition and weight of the transaction. As transactions
+compete by feerate [fee/weight], a lighter transaction requires a lower
+fee to reach the same feerate.
+
+Some wallets, such as the Bitcoin Core wallet, try to combine inputs
+such that they avoid needing a change output altogether. Avoiding change
+saves the weight of an output now, but also saves the future cost of
+spending the change output later. Unfortunately, such input combinations
+will only seldom be available unless the wallet sports a large UTXO pool
+with a broad variety of amounts.
+
+Modern output types are more blockspace-efficient than older output
+types. E.g. spending a P2TR input incurs less than 2/5ths of a P2PKH
+input’s weight. (Try it with our [transaction size
+calculator][]!) For multisig wallets, the recently
```

Comment:
```suggestion
calculator][]) For multisig wallets, the recently
```

Reply:
That’s a whole sentence in the parentheses, so wouldn’t it need punctuation?